interface/src/app/gene/network/network.js

```diff
@@ -40,7 +41,7 @@ angular.module('adage.gene.network', [
   });
 }])
 
-.factory('EdgeService', ['$resource', 'ApiBasePath',
+.factory('Edge', ['$resource', 'ApiBasePath',
```

Comment:
I feel that having "Service" at the end of service name is redundant. I saw the usage of both. I don't know which one is better, but I prefer a shorter name.